projects/packages/blaze/src/class-blaze.php

```diff
@@ -61,22 +61,22 @@ public function register() {
 	 *
 	 * @return array
 	 */
-	public function jetpack_promote_posts_row_action( $post_actions, $post ) {
+	public function jetpack_blaze_row_action( $post_actions, $post ) {
 		$post_id = $post->ID;
 
 		// @todo wrap in method call, for general use?
 		$promote_post_url = Redirect::get_url(
```

Comment:
```suggestion
		$blaze_url = Redirect::get_url(
```

Reply:
Reminds me the redirect-url needs updating in wpcom.